k9mail/src/main/res/layout/openpgp_encrypt_description_dialog.xml

```diff
@@ -63,7 +63,7 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_gravity="center_vertical"
-            android:src="@drawable/ic_action_next_message_light"
+            android:src="?attr/iconActionNextMessage"
```

Comment:
This should be a separate attribute with an appropriate name. Semantically it's different from 'next message'.

Reply:
Done